dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml

```diff
@@ -76,6 +76,11 @@
       <artifactId>cloudant-http</artifactId>
       <version>2.8.0</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+      <version>2.11.2</version>
```

Comment:
For all of the version changes, I think you just need to change the versions instead of duplicating the dependencies.

Reply:
There are components in CL that still depend on the `com.ibm.ws.com.fasterxml.jackson` project... which pulls in version `2.9.1` of the relevant Jackson libraries. We cannot remove them until we have updated the components in CL to point at the `2.11.2` version of these libraries... and this cannot be done until they are in OL master. So these changes need to be made in phases, requiring 3 PRs in total... of which this is just the first PR.